lib/app.rb

```diff
@@ -1,35 +1,68 @@
+require 'finder'
+require 'creator'
+
 class App
   attr_reader :contacts
 
-  def initialize(ui)
+  def initialize(ui, kernel = Kernel)
     @ui = ui
     @contacts = []
+    @kernel = kernel
   end
 
   FIELDS = [:first_name, :last_name, :email_address, :mobile_number, :twitter_handle]
```

Comment:
I guess I was trying to put FIELDS into a central place where it's easy to find. 
To remove or add a field, you only need to modify the FIELDS array. 
Would it be better create something like a config file that has this data? 
My logic was the same with MENU_OPTIONS.